pkg/cmd/issue/view/view.go

```diff
@@ -161,22 +196,102 @@ func printHumanIssuePreview(io *iostreams.IOStreams, issue *api.Issue) error {
 	}
 
 	// Body
-	if issue.Body != "" {
-		fmt.Fprintln(out)
-		style := markdown.GetStyle(io.TerminalTheme())
-		md, err := markdown.Render(issue.Body, style, "")
+	fmt.Fprintln(out)
+	if issue.Body == "" {
+		issue.Body = "_No description provided_"
+	}
+	style := markdown.GetStyle(io.TerminalTheme())
+	md, err := markdown.Render(issue.Body, style, "")
+	if err != nil {
+		return err
+	}
+	fmt.Fprint(out, md)
+	fmt.Fprintln(out)
+
+	// Comments
+	if issue.Comments.TotalCount > 0 {
+		comments, err := issueComments(io, issue.Comments)
 		if err != nil {
 			return err
 		}
-		fmt.Fprintln(out, md)
+		fmt.Fprint(out, comments)
 	}
-	fmt.Fprintln(out)
 
 	// Footer
-	fmt.Fprintf(out, cs.Gray("View this issue on GitHub: %s\n"), issue.URL)
+	fmt.Fprintf(out, cs.Gray("View this issue on GitHub: %s"), issue.URL)
+
 	return nil
 }
 
+func issueComments(io *iostreams.IOStreams, comments api.IssueComments) (string, error) {
+	var b strings.Builder
+	cs := io.ColorScheme()
+	retrievedCount := len(comments.Nodes)
+	hiddenCount := comments.TotalCount - retrievedCount
+
+	if hiddenCount > 0 {
+		fmt.Fprint(&b, cs.Gray(fmt.Sprintf("———————— Hiding %v comments ————————", hiddenCount)))
```

Comment:
I used the em dash here since it creates a nicer line than the normal short dash. I was worried that it being a unicode character would be an issue but it appears to have wide support across terminals. Also, we are now rendering emoji reactions which are also unicode characters, so it seemed reasonable to continue to use them. In the future it might be nice to have an option to toggle unicode characters for users who have terminals that do not support them.

Reply:
👏 the use of em dash

I found it weird to see "Hiding N comments" in a text-only interface where toggling those comments on/off isn't immediately available like it is on a web page when something is collapsed/hidden. Would it be a better alternative to say "Not showing N comments" instead to drive the point that the comments were not "hidden" (which often implies some reversibility), but omitted from rendering altogether?

BTW, when only 1 comment is hidden, the sentence should say `1 comment`. You can use our `Pluralize` helper to automatically render `comment/comments` based on the number provided.